core/backend/src/SchemaUtility.ts

```diff
@@ -0,0 +1,47 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module SchemaUtility
+ */
+
+import { IModelJsNative } from "@bentley/imodeljs-native";
+import { BentleyStatus } from "@itwin/core-bentley";
+import { IModelError } from "@itwin/core-common";
+import { ECSchemaXmlContext } from "./ECSchemaXmlContext";
+
+/**
+ * Utility class for EC Schemas
+ */
+export class SchemaUtility {
```

Comment:
Why is this a class? It has no data members. Free functions or namespace would make more sense.
Please avoid making "Utility" class/namespaces - they become disorganized dumping grounds.